ffi/src/lib.rs

```diff
@@ -211,18 +377,46 @@ impl From<String> for Value {
     }
 }
 
+// WARNING: This should only be called with values >= 1.
+// In much of the Go code, v.len == 0 is used to indicate a null-terminated string.
+// This may cause a panic!
+impl From<u32> for Value {
+    fn from(v: u32) -> Self {
+        Self {
```

Comment:
debug_assert if you prefer
```suggestion
        assert_ne!(v, 0);
        Self {
```

Reply:
I'll use the real assert. If this happens, it could cause all sorts of issues - might be best to notice it at the source